api/src/routes/auth.ts

```diff
@@ -1,7 +1,7 @@
 import { Express, NextFunction, Request, Response } from "express";
 import * as ExpressSession from "express-session";
 
-import { AuthUser } from "../data";
+import { AuthUser } from "../data/models";

```

Comment:
@datajohnson I think this change got lost in a merge somewhere, as it is no longer in the `development` branch.